README.md

```diff
@@ -803,10 +809,65 @@ Thanks also to
 [@chrishtr](https://github.com/chrishtr),
 [@dvoytenko](https://github.com/dvoytenko),
 [@housseindjirdeh](https://github.com/housseindjirdeh),
-[@jakearchibald](https://github.com/jakearchibald), and
+[@jakearchibald](https://github.com/jakearchibald),
+[@matt-buland-sfdc](https://github.com/matt-buland-sfdc), and
 [@slightlyoff](https://github.com/slightlyoff)
 for their help in exploring this space and providing feedback.
 
+## Appendix: types of navigations
+
+The web platform has many ways of initiating a navigation. For the purposes of this API, and in particular the [`navigate` event](#navigation-monitoring-and-interception), the following is intended to be a comprehensive list:
+
+- Users triggering navigations via browser UI, including (but not necessarily limited to):
```

Comment:
Below are mechanisms, rather than actions. A user uses one of these to initiate a navigation. To maintain the current description, I'd expect the list to contain items like "change the text in the URL bar" (to initiate a navigation).

Or, describe these as "mechanisms that can be used to trigger a navigation" (no longer using active voice)

Reply:
The below are specific pieces of browser UI, so I think the phrasing is appropriate here? i.e. "... browser UI, including (but not necessarily limited to): the URL bar, ..."

Reply:
Yeah, I see. Maybe just change "triggering" to "can trigger" then?